execexam/display.py

```diff
@@ -39,46 +39,50 @@ def display_tldr(console: Console) -> None:
         "[bold red]Please ensure you are in the directory with the pyproject.toml file to run these commands.[/bold red]\n"
```

Comment:
Here is the line @rebekahrudd and I are referring to with the red output.

Reply:
I get how this could be confusing! I just removed it, could you please check it out again? Thanks!